### PR TITLE
Create forgot password view

### PR DESCRIPTION
**Pull request by rem1niscence**

### CODE REVIEW

File: frontend/src/views/Core/ForgotPassword/ForgotPassword.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+class ForgotPassword extends Component {
+  render() {
+    return (
+      <Container fluid id={"forgot-password-page"}>
+        <Row>
+          <Col id={"navbar"}>
+            <img src={"/logo.png"} alt="logo" id={"main-logo"} />
+
+            <div className={"help"}>
```
Comment (xerp) [inline frontend/src/views/Core/ForgotPassword/ForgotPassword.js]: We need to create a Help component, please create one and reuse here and the Login Page.
Create this component inside core/components.

File: frontend/src/views/Core/ForgotPassword/ForgotPassword.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@import "src/scss/Core";
+
+#forgot-password-page {
+  #navbar {
```
Comment (xerp) [inline frontend/src/views/Core/ForgotPassword/ForgotPassword.scss:2]: As you said, please create a **sidebar** component and move the css of these components to **Core** scss
Create the sidebar inside _core/components_ for the js and for css move to _scss/Core.scss_.

File: frontend/src/views/Core/ForgotPassword/ForgotPassword.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
+@import "src/scss/Core";
```
Comment (xerp) [inline frontend/src/views/Core/ForgotPassword/ForgotPassword.scss]: In the css of each component page, _we need only the css for the page component_, for general components we need the logic on Core.scss

File: frontend/src/_routes.js [hunk 4]
```diff
@@ -1,17 +1,20 @@
 import Login from "./views/Core/Login/Login";
 import Dashboard from "./views/Core/Dashboard/Dashboard";
+import ForgotPassword from "./views/Core/ForgotPassword/ForgotPassword";
 import GithubAuthProviderHook from "./core/components/providers/auth/GithubAuthProviderHook";
 import GoogleAuthProviderHook from "./core/components/providers/auth/GoogleAuthProviderHook";
 
 export const LOGIN_PATH = "/login";
 export const HOME_PATH = "/dashboard";
+export const FORGOTPASSWORD_PATH = "/forgot-password";
```
Comment (xerp) [inline frontend/src/_routes.js]: For these ones, please refactoring to a JSON variable named routePaths, with the following structure:

```javascript
export const routePaths = {
login: "/login",
home: "/home",
forgot_password: "/forgot-password"
.
.
.
};
```
Then change all old references for the new ones.

**Review by xerp — CHANGES_REQUESTED** [review]
@justaname94 Please check the comments on changes in this PR.

**Comment by xerp** [comment]
Also for our conversation, we need a naming convention for branch, we will use the following prefixes:

- For **features**: _feature/[frontend|backend], feature/[frontend|backend]/[page|component]_
- For **bugs**: _bugfix/[frontend|backend], bugfix/[frontend|backend]/[page|component]_

Examples:

1. We need to create app page, the branch name will be: feature/frontend/page/app.
2. We need to resolve an issue about balance on backend pokt service, the branch name will be: bugfix/backend/[name]

**Comment by rem1niscence** [comment]
All right, I'll be applying them tomorrow morning

File: frontend/src/core/components/Help/Help.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  return (
+    <div className={"help"}>
+      <Button variant={"link"}>
+        <FontAwesomeIcon icon={faQuestionCircle} />
+      </Button>
+    </div>
+  );
+};
+
+export default Help;
```
Comment (xerp) [inline frontend/src/core/components/Help/Help.js]: For component, we need to follow class based component style.
Please use _core/components/AuthProviderButton.js_ as example.

Also use propTypes property of component if is needed.

**Review by xerp — CHANGES_REQUESTED** [review]

**Comment by xerp** [comment]
Looks good

**Pull request merged**